### ISSUE TITLE
A coding question for Reppoint detector: points2bbox transform_method = 'moment'

### ISSUE THREAD

**Issue by xuyongzhi** [open-mmlab/mmdetection#2019]
In mmdet/models/anchor_heads/reppoints_head.py  Line 200:
```
moment_transfer = (self.moment_transfer * self.moment_mul) + (
                self.moment_transfer.detach() * (1 - self.moment_mul))
```
Why it needs to use ```self.moment_transfer.detach() * (1 - self.moment_mul)``` instead of directly using ```self.moment_transfer * (1 - self.moment_mul)```.

**Comment by hellock** [comment]
This line is used to reduce the learning rate by `self.moment_mul`.

**Comment by xuyongzhi** [comment]
Cool. Thanks!
